cmd/index_add.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2022 The Falco Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/falcosecurity/falcoctl/pkg/index"
+	"github.com/falcosecurity/falcoctl/pkg/options"
+)
+
+type indexAddOptions struct {
+	*options.CommonOptions
+}
+
+func (o *indexAddOptions) Validate(args []string) error {
+	if _, err := os.Stat(falcoctlPath); os.IsNotExist(err) {
+		err = os.Mkdir(falcoctlPath, 0o700)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// NewIndexAddCmd returns the index add command.
+func NewIndexAddCmd(ctx context.Context, opt *options.CommonOptions) *cobra.Command {
+	o := indexAddOptions{
+		CommonOptions: opt,
+	}
+
+	cmd := &cobra.Command{
+		Use:                   "add [NAME] [URL] [flags]",
+		DisableFlagsInUseLine: true,
+		Short:                 "Add an index to the list of indexes",
+		Long:                  "Add an index to the list of indexes. Indexes are used to perform search operations for artifacts",
```

Comment:
It might be unclear to the user what a "list of indexes" is.

What about `Add an index to the falcoctl local configuration` ?